magma/backend/coreir_utils.py

```diff
@@ -138,7 +138,7 @@ def is_clock_or_nested_clock(p):
     if issubclass(p, Array):
         return is_clock_or_nested_clock(p.T)
     if issubclass(p, Tuple):
-        for item in p.Ts:
+        for item in p.types():
```

Comment:
This is an old `hwtypes` branch related interface change that wasn't caught until now (this is an error code path, so doesn't show up in normal function)